packages/flame/test/geometry/shape_intersections_test.dart

```diff
@@ -0,0 +1,302 @@
+import 'package:flame/collisions.dart';
+import 'package:flame/components.dart';
+import 'package:flame/geometry.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('CircleCircleIntersections', () {
```

Comment:
There should be a file where these tests can be incorporated, where the rest of the intersection tests are. Since we have another concept of shapes currently too it gets a bit confusing if this file is named shape tests.